### ISSUE TITLE
MIMO and MISO final plots

### ISSUE THREAD

**Issue by david1412** [david1412/Time-varying--MIMO#9]
I have simulated the MIMO average system distance.
![mimomi1](https://user-images.githubusercontent.com/30881405/35018311-276d57fe-fb21-11e7-806e-65abee410415.png)
![mimomi2](https://user-images.githubusercontent.com/30881405/35018314-2a0a020a-fb21-11e7-8fce-8c5dafb0c516.png)

 I got this plots . I havent updated the code in the github . I share the plotting here

**Comment by david1412** [comment]
please check the new plots after some modification based on your comments.

![4](https://user-images.githubusercontent.com/30881405/35109290-360c17e0-fc76-11e7-87c9-d50f11b87d67.png)

![3](https://user-images.githubusercontent.com/30881405/35109287-33c46622-fc76-11e7-8bc5-39c363394685.png)

![2](https://user-images.githubusercontent.com/30881405/35109286-31bfc2d6-fc76-11e7-9abd-fb567ac84107.png)

![1](https://user-images.githubusercontent.com/30881405/35109278-2ecbda6a-fc76-11e7-8482-88547a0a651f.png)

**Comment by narahahn** [comment]
Would be better to run the simulations for more different values of `Omega`, about 10?

**Comment by david1412** [comment]
i have checked till omega 6 ..  i will check the omegas till 10 .. to compute the 
undersampling ( it took omega= 6)
,critical sampling( omega=4)
 and over sampling ( omega=1) i had the results ... beacause the anti-aliasing condition foe the computation i got ( 4.56) ... i will check more ... I hope i reached the state of Art

**Comment by narahahn** [comment]
I don't mean the value of `Omega` but the number of simulations you run for different values of `Omega`. Now you have only 3.

**Comment by david1412** [comment]
yes i have only 3 and saved the plots to keep in the thesis book ... i checked different values for omega . if i take all the plots in the thesis book .. it is page count will be large .. 

apart from upsampling,under,critical ... i will run till and save in the file. 

should i keep all the different value for Omega ? ( 1~10) in the thesis

**Comment by narahahn** [comment]
Three plots will suffice for system distance (phi vs system distance).
But in order to show the change of the average system distance for varying angular speed (Omega vs average system distance), more results would be much better.
It doesn't mean that you should include also the system distance plots.

**Comment by david1412** [comment]
1 . first point i will keep three plots 
2. I will run more tests and keep more results for ( omega vs averages system distance). 
3. I understood the point and follow your suggestion.
